src/substrait-to-d3.js

```diff
@@ -0,0 +1,296 @@
+'use strict';
+
+const { SubstraitParser } = require('./substrait-parser');
+const d3 = require('d3');
+
+// Populating map with nodes information
+function populateNodes(plan, nodes) {
```

Comment:
Maybe rename to something like `createNodeIdToNodeMap`?

Reply:
Made the change, thanks!